apps/champions/test/battle_test.exs

```diff
@@ -1681,4 +1677,179 @@ defmodule Champions.Test.BattleTest do
                Champions.Battle.Simulator.run_battle([speeder], [damager], maximum_steps: maximum_steps + 1).result
     end
   end
+
+  describe "Initial Modifiers" do
+    test "Initial Modifiers modify stats correctly" do
+      # We will run a battle with a unit that has some modifications to its stats.
+      # Its initial state for "health" would have been 5 due to its character's stats, but it will be buffed up to 10.
+      # It will fight a unit that will deal 5 damage to it. Without the modifications, it would die.
+      # After surviving an attack, it will deal 100% ATK damage to the enemy, that also has 10 health.
+      # Because of the attack buff, which brings hist total ATK to 10, he will kill the enemy in one hit, and battle won't end on a timeout.
```

Comment:
```suggestion
      # Because of the attack buff, which brings his total ATK to 10, he will kill the enemy in one hit, and battle won't end on a timeout.
```